Source/Extensions/UIBarStyle+stringValue.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  UIBarStyle+stringValue.swift
+//  Frames
+//
+//  Created by Harry Brown on 24/03/2022.
+//  Copyright © 2022 Checkout. All rights reserved.
+//
+
+import UIKit
+
+extension UIBarStyle {
+    var stringValue: String {
+        switch self {
+        case .default:
+            return "default"
+        case .black, .blackOpaque:
+            return "black"
+        case .blackTranslucent:
+            return "blackTranslucent"
```

Comment:
You can move all these hardcoded strings into a enum constants which can reused elsewhere